systers_portal/community/models.py

```diff
@@ -134,11 +140,17 @@ def set_new_admin(self, new_admin):
         return OK
 
 
+@receiver(pre_save, sender=Community)
+def set_slug_community(sender, instance, *args, **kwargs):
+    """Automatically generate slug from community name"""
+    instance.slug = unique_slug_generator(instance)
+
+
 class RequestCommunity(models.Model):
     """Model to represent new community requests"""
     name = models.CharField(
         max_length=255, verbose_name="Proposed Community Name")
-    slug = models.SlugField(max_length=150, unique=True, verbose_name="Slug")
+    slug = models.SlugField(max_length=150, unique=True, verbose_name="Slug", blank="True")
```

Comment:
Also here.

Reply:
@aviaryan these updates are being made locally and tested... When all tests are passed the changes shall be reflected here.